test/runtests.jl

```diff
@@ -38,10 +38,10 @@ end
 @testitem "KomaUI" tags=[:koma] begin
 
     using Blink
+    # Opens KomaUI
 
-    # Opens UI
-    w = KomaUI(return_window=true) 
-
+    @unsafe_blink
```

Comment:
Yo actually fix the bug (opposed to only fix the tests) the call to the macro needs to be inside the KomaUI function.